lancet/commands/configuration.py

```diff
@@ -89,7 +130,8 @@ def init(ctx, force):
         )
         ctx.exit(1)
 
-    project_key = click.prompt("Project key on the issue tracker")
+    project_id = click.prompt("Project id on Gitlab")
```

Comment:
Lancet supports gitlab and jira, avoid hardcoding specific services.

Reply:
WIP

Reply:
However, I need to check the generations for gitlab and jira once checking and adapting these commands. The scr-manager probably also need to be updated depending on the choice. For now the important one is the test to check the expected outcome. I expect the setup command to evolve the deeper we dig.